temporal-sdk/src/main/java/io/temporal/internal/worker/Poller.java

```diff
@@ -316,4 +317,15 @@ private static void shutdownAndAwaitTermination(ExecutorService pool) {
       Thread.currentThread().interrupt();
     }
   }
+
+  private static boolean shouldIgnoreDuringTermination(Throwable ex) {
+    // if we are terminating and getting rejected execution - it's normal
+    if (ex instanceof RejectedExecutionException) return true;
```

Comment:
Not the best code style, but an attempt to do
```
    return
      // if we are terminating and getting rejected execution - it's normal
      ex instanceof RejectedExecutionException ||
      // if the worker thread gets InterruptedException - it's normal during shutdown
      ex instanceof InterruptedException ||
      // if we get wrapped InterruptedException like what PollTask or GRPC clients do with setting
      // Thread.interrupted() on - it's normal during shutdown too. See PollTask javadoc.
      ex.getCause() instanceof InterruptedException;
```
was butchered by autoformatter.